src/eclingo/parsing/transformers/parser_negations.py

```diff
@@ -44,12 +45,15 @@ def visit_UnaryOperation(self, x):
         external = x.argument.external
 
         if self.reification:
-            aux_name = "-" + name
+            aux_name = name
         else:
             aux_name = self.strong_negation_prefix + "_" + name
             self.replacement.add((name, len(arguments), aux_name))
 
         atom = ast.Function(location, aux_name, arguments, external)
+        if self.reification:
```

Comment:
I've finally solved that issue when using reification. The problem was the declaration of the name "-a" of the Function when it should have been an UnaryOperation that had an ast.Function as argument with the name "a".